content/en/docs/Concepts/olm-architecture/operator-catalog/creating-an-update-graph.md

```diff
@@ -255,6 +255,308 @@ entries:
       - myoperator.v0.3.0
 ```
 
+## Skip vs. SkipRange vs. Replaces by Example
+
+For these examples we will be considering an operator that is already installed (`Installed`) and how update options are selected for it.
+
+```mermaid
```

Comment:
Have you looked at the deploy target in the CI? This graph does not render as a graph at all.